### PR TITLE
AboutDialog improvements

### PR DESCRIPTION
**Pull request by iam-TJ**
This is a combination of three individual changes; two fixes and one feature.

In combination they improve the AboutDialog experience.

1. ensuring that `Label` components correctly wrap in the ThirdPartyLicenses tab.
2. correcting the spelling of "License" to "Licence" (the commit message points to [this advice](http://english-grammar-lessons.co.uk/easily_confused/licence_license.htm).
3. allowing the dialog to scale to 75% of the application window instead of being forced to 640x400.

If the latter scalable dialog is acceptable I intend improving other dialogs where the same fixed size causes truncated content, especially in error message dialogs as I mentioned in issue #524.

### CODE REVIEW

**Comment by hipersayanX** [comment]
I think both cases for [license](https://www.merriam-webster.com/dictionary/license) and [licence](https://www.merriam-webster.com/dictionary/licence) are ok, but [license](https://github.com/webcamoid/webcamoid/blob/master/COPYING) is more commonly used, also its looks weird to write a **c (s)**, after another **c (s)**, there is a grammar rule for that but I don't remember the name.

**Comment by iam-TJ** [comment]
"License" is only common in as much as it is used in the USA and that country's culture and language tends to infect elsewhere. Because many software licences are created in the USA that spelling tends to dominate. 

In countries with their own English language history, and for 'Business English', the British/Commonwealth spelling and grammar is the commonly accepted preference.

Note the Merrian-Webster is a USA publisher. The 'oracle' for English language is the Oxford English Dictionary (OED) :)

**Comment by hipersayanX** [comment]
I don´t want to enter into a culture war, I'll continue using **license** as in **GNU General Public License**, the same way as I'll continue using **master** instead of **main**.

**Comment by iam-TJ** [comment]
I've removed the "licence" changes from this PR so it only contains:
```
$ gitlog -n 6                                                             
21f0812c 2022-06-01 11:04:20 +0100 N Tj Merge branch 'feature-aboutdialog-size-scaling' into pr-aboutdialog-improvements
51c253d2 2022-05-31 16:57:27 +0100 N Tj Merge branch 'fix-about-thirdpartylicences-truncated-text' into pr-aboutdialog-improvements
4f2da062 2022-05-31 16:50:32 +0100 N Tj AboutDialog: allow dialog to scale with application window                    
7538b07f 2022-05-31 16:27:01 +0100 N Tj ThirdPartyLicences: make truncated text wrap
f25b8b1c 2022-05-19 07:19:12 +0200 E AHOHNMYC Translated using Weblate (Russian) 
```
I can handle the "licence" issue in a en_GB translation :)

File: StandAlone/share/qml/AboutDialog.qml [hunk 1]
```diff
@@ -54,13 +54,13 @@ Dialog {
             }
             TabButton {
                 //: Program license.
-                text: qsTr("License")
+                text: qsTr("Licence")
```
Comment (hipersayanX) [inline StandAlone/share/qml/AboutDialog.qml:57]: @iam-TJ can you remove this changes before merging, please?

File: StandAlone/share/qml/AboutDialog.qml [hunk 2]
```diff
@@ -54,13 +54,13 @@ Dialog {
...
             }
             TabButton {
                 /*: License for 3rd party components used in Webcamoid, like
                     libraries and code snippets.
                  */
-                text: qsTr("3rd Party Licenses")
+                text: qsTr("3rd Party Licences")
```
Comment (hipersayanX) [inline StandAlone/share/qml/AboutDialog.qml:63]: And, this one.

Comment (iam-TJ) [reply StandAlone/share/qml/AboutDialog.qml]: Something weird going on with github! My local branch (which I force pushed to my github) does not contain those "License" changes - those were all contained in a 3rd commit which I dropped from the branch. Force-pushing once more now!
```
$ git status -u no                                                                                                                                                                              
On branch pr-aboutdialog-improvements                                                                                                                                                                                                       
nothing to commit, working tree clean                                                                                                                                                                                                       
tj@sunny:/srv/NAS/Sunny/SourceCode/webcamoid$ gitlog -n 5  
21f0812c 2022-06-01 11:04:20 +0100 N Tj Merge branch 'feature-aboutdialog-size-scaling' into pr-aboutdialog-improvements                                                                                                                    
51c253d2 2022-05-31 16:57:27 +0100 N Tj Merge branch 'fix-about-thirdpartylicences-truncated-text' into pr-aboutdialog-improvements                                                                                                         
4f2da062 2022-05-31 16:50:32 +0100 N Tj AboutDialog: allow dialog to scale with application window                    
7538b07f 2022-05-31 16:27:01 +0100 N Tj ThirdPartyLicences: make truncated text wrap                                  
f25b8b1c 2022-05-19 07:19:12 +0200 E AHOHNMYC Translated using Weblate (Russian)
$ git diff f25b8b1c                                                       
diff --git a/StandAlone/share/qml/AboutDialog.qml b/StandAlone/share/qml/AboutDialog.qml                              
index 2701f2e5..65576838 100644                                                                                       
--- a/StandAlone/share/qml/AboutDialog.qml                                                                                                                                                                                                  
+++ b/StandAlone/share/qml/AboutDialog.qml                                                                            
@@ -26,8 +26,8 @@ import Webcamoid 1.0                                                                                
 Dialog {                                         
     id: aboutDialog                                                                                                  
     standardButtons: Dialog.Close                                                                                    
-    width: AkUnit.create(640 * AkTheme.controlScale, "dp").pixels                                                    
-    height: AkUnit.create(400 * AkTheme.controlScale, "dp").pixels
+    width: wdgMainWidget.width * 0.75                     
+    height: wdgMainWidget.height * 0.75                                                                              
     modal: true                                                                                                      
     title: qsTr("About %1").arg(mediaTools.applicationName)                                                          
                                                                                                                      
diff --git a/StandAlone/share/qml/ThirdPartyLicenses.qml b/StandAlone/share/qml/ThirdPartyLicenses.qml                
index b691ac76..ee17ca1e 100644                                                                                       
--- a/StandAlone/share/qml/ThirdPartyLicenses.qml                                                                                                                                                                                           
+++ b/StandAlone/share/qml/ThirdPartyLicenses.qml                                                                                                                                                                                           
@@ -37,11 +37,13 @@ Page {                                                                                            
                 text: "The followings are the Licenses for 3rd-party work incorporated into Webcamoid. <b>These Licenses DOES NOT applies to Webcamoid itself.</b>"                                                                        
                 wrapMode: Text.WordWrap                                                                              
                 Layout.fillWidth: true                                                                               
+                Layout.maximumWidth: parent.width                                                                                                                                                                                          
             }                                                                                                        
             Label {                                                                                                  
                 text: "License for resources taken from openclipart.org:"                                            
                 wrapMode: Text.WordWrap                                                                              
                 Layout.fillWidth: true                                                                               
+                Layout.maximumWidth: parent.width                                                                    
             }                                                                                                        
             TextArea {                                    
                 text: mediaTools.readFile(":/Webcamoid/share/3rd-party/licenses/openclipart.txt")                    
@@ -52,6 +54,7 @@ Page {                
                 text: "License for code taken from OpenCV:"                                                          
                 wrapMode: Text.WordWrap                                                                              
                 Layout.fillWidth: true                    
+                Layout.maximumWidth: parent.width                                                                    
             }                                                                                                        
             TextArea {                                                                                               
                 text: mediaTools.readFile(":/Webcamoid/share/3rd-party/licenses/OpenCV.txt")                         
@@ -62,6 +65,7 @@ Page {              
                 text: "License for code and algorithms used in Temperature plugin:"                                  
                 wrapMode: Text.WordWrap                                                                                                                                                                                                    
                 Layout.fillWidth: true                                                                                                                                                                                                     
+                Layout.maximumWidth: parent.width                                                                    
             }                                                                                                        
             TextArea {                                                                                               
                 text: mediaTools.readFile(":/Webcamoid/share/3rd-party/licenses/TemperatureAlgorithm.txt")           
@@ -72,6 +76,7 @@ Page {                                                                                              
                 text: "License for the usb.ids file:"                                                                
                 wrapMode: Text.WordWrap                                                                              
                 Layout.fillWidth: true                                                                               
+                Layout.maximumWidth: parent.width                                                                    
             }                          
             TextArea {                
                 text: mediaTools.readFile(":/Webcamoid/share/3rd-party/licenses/UsbIds.txt")
```

**Comment by hipersayanX** [comment]
Thanks!

**Pull request merged**